cmd/eksctl-anywhere/cmd/getpackage.go

```diff
@@ -0,0 +1,67 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
+
+	"github.com/aws/eks-anywhere/pkg/executables"
+	"github.com/aws/eks-anywhere/pkg/features"
+)
+
+type getPackageOptions struct {
+	output string
+}
+
+var gpo = &getPackageOptions{}
+
+func init() {
+	getCmd.AddCommand(getPackageCommand)
+	getPackageCommand.Flags().StringVarP(&gpo.output, "output", "o", "", "Specifies the output format (valid option: json, yaml)")
+}
+
+var getPackageCommand = &cobra.Command{
+	Use:     "package(s) [flags]",
+	Aliases: []string{"package", "packages"},
+	Short:   "Get package(s)",
+	Long:    "This command is used to display the installed packages",
+	PreRunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
Let's move this to a separate function like we do here: https://github.com/aws/eks-anywhere/blob/main/cmd/eksctl-anywhere/cmd/supportbundle.go#L80